### PR TITLE
feat: checklist menu item for 5.2

### PR DESCRIPTION
**Pull request by tylerkelly13**

### CODE REVIEW

File: advanced/editor-control-identifiers.md [hunk 1]
```diff
@@ -84,9 +84,11 @@ tinyMCE.activeEditor.ui.registry.getAll().buttons
...
-| numlist              | [lists]({{ site.baseurl }}/plugins/lists/)                                                                                         | Formats the current selection as a numbered list.                                     |
-| media                | [media]({{ site.baseurl }}/plugins/media/)                                                                                         | Creates/Edits embedded media elements.                                                |
-| nonbreaking          | [nonbreaking]({{ site.baseurl }}/plugins/nonbreaking/)                                                                             | Inserts a nonbreaking space into the editor.                                          |
+| numlist              | [lists]({{ site.baseurl }}/plugins/lists/)                                                                                         | Formats the current selection as a numbered list.                                    * Used in quality assurance of software engineering to check process compliance, code standardization, and error prevention.
```
Comment (lnewson) [inline advanced/editor-control-identifiers.md]: Ahh what?

Comment (tylerkelly13) [reply advanced/editor-control-identifiers.md]: fixed

File: enterprise/checklist.md [hunk 2]
```diff
@@ -7,23 +7,25 @@ keywords: lists todo checklist
...
-*   **Industry applications** - There are a lot of professional applications of using a Checklist, such as:
-    *   Used in quality assurance of software engineering to check process compliance, code standardization, and error prevention.
-    *   Pre-flight checklists to aid in aviation safety to ensure that critical items are not overlooked.
-    *   Used by some investors as a critical part of their investment process.
-    *   Checklists are useful for school camps to create emergency survival kits.
-    *   In professional diving checklists can be used for preparation of equipment for a diving session.
+* **Private & Work** - Checklist can help in keeping work and private tasks separate for more efficiency.
+* **Tracking Bills** - Checklists can be created to avoid forgetting payment cycles to pay a bill on time.
+* **Shopping list** - Checklist can help create a grocery list in the browser that can be shared with the other family members to collaborate or act as a reminder while shopping.
+* **Entertainment tracker** - Checklist can help create list of movies & TV series to keep a tab on the movies to watch or set weekly reminders at the right time to avoid missing a show.
```
Comment (ltrouton) [inline enterprise/checklist.md]: Should this be 'create **a** list of movies'?

Comment (tylerkelly13) [reply enterprise/checklist.md]: lol probably. I only fixed the markup in this section

File: _includes/codepens/checklist/index.js [hunk 3]
```diff
@@ -2,5 +2,9 @@ tinymce.init({
   selector: "textarea#checklist",
   plugins: "lists checklist",
   toolbar: "checklist",
-  height: "600px",
+  height: 200,
+  width: 350,
+  menu: {
+	  format: { title: 'Format', items: 'checklist | bold italic underline strikethrough superscript subscript codeformat | formats blockformats fontformats fontsizes align | forecolor backcolor | removeformat' }
```
Comment (lnewson) [inline _includes/codepens/checklist/index.js]: This tab character probably shouldn't be here.

File: release-notes/release-notes52.md [hunk 4]
```diff
@@ -82,6 +82,22 @@ For an updated list of {{site.productname}} icons, see: [Editor icon identifiers
 
 The following premium plugins have been updated for {{site.productname}} 5.2.
 
+### Accessibility Checker
```
Comment (lnewson) [inline release-notes/release-notes52.md]: This is missing a couple things if you're concerned? 
- We added the new check for checking duplicate ids
- We improved the checking around decorative images.

**Pull request merged**